controller/array_action/ds8k_rest_client.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, service_address, user, password,
         client_kwargs = {'service_address': service_address,
                          'user': user,
                          'password': password,
+                         'verify': False,
```

Comment:
do we have a ticket to support verification?

Reply:
do we want to?

Reply:
sure

Reply:
can you open a ticket?

Reply:
CSI-4103